src/components/views/rooms/EventTile.js

```diff
@@ -290,6 +290,18 @@ module.exports = WithMatrixClient(React.createClass({
 
         var left = 0;
 
+        var readReceiptData = Object.create(null);
+        var room = this.props.matrixClient.getRoom(this.props.mxEvent.getRoomId());
+        if (room) {
+            // [ {type/userId/data} ]
+            room.getReceiptsForEvent(this.props.mxEvent).forEach(function(r) {
```

Comment:
I'm not terribly happy about this being here as well as `this.props.readReceipts`. It gives a very odd interface to this component, which is becoming less and less 'view'-like.

Can we push this up into `MessagePanel` and add the timestamp data to `this.props.readReceipts`?